src/main/java/io/vertx/core/impl/VertxFactory.java

```diff
@@ -159,6 +161,14 @@ private VertxTracer createTracer() {
     return tracer;
   }
 
+  private ContextManager createContextManager() {
```

Comment:
I don't store anything in the factory so that the context manager doesn't share the same context manager across many vert.x instances, which would leak its stateful nature. One vert.x instance, one context manager. I explored adding here a context manager factory but looked more verbose and I didn't think it was useful to add another layer of indirection.